styleguide-app/Examples/IconHelpers.elm

```diff
@@ -0,0 +1,47 @@
+module Examples.IconHelpers exposing (viewIconSection)
```

Comment:
💭 can we name this without the word "helper"?  It seems like "IconExample" would be a reasonable name here, since it seems like it has two functions for displaying icon examples.

Reply:
Love it!